packages/terra-horizontal-divider/src/HorizontalDivider.scss

```diff
@@ -0,0 +1,9 @@
+:local {
+  .divider {
+    border-color: var(--terra-horizontal-divider-border-color, #b2b5b6);
+    border-style: solid;
+    border-top-width: var(--terra-horizontal-divider-border-width, 1px);
```

Comment:
Sorry, I looked at this again and it look like you'll need to explicitly set the other border widths (left, right, bottom) to be 0. The divider is still rendering as a 2px line since the other borders are defaulted by the browser for hr elements.

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/66750538e3081801d6a4e796d34466a70c963a68#diff-4a12721d904b23df9119993e41aa2810R1